harmony/harmony.py

```diff
@@ -269,16 +279,17 @@ def _submit_url(self, request: Request) -> str:
         variables = [v.replace('/', '%2F') for v in request.variables]
         vars = ','.join(variables)
         return (
-            f'https://{self.config.harmony_hostname}/{request.collection.id}'
+            f'{self.config.root_url}'
+            f'/{request.collection.id}'
             f'/ogc-api-coverages/1.0.0/collections/{vars}/coverage/rangeset'
         )
 
-    def _status_url(self, job_id: str) -> str:
+    def _status_url(self, job_id: str, link_type: LinkType = LinkType.https) -> str:
         """Constructs the URL for the Job that is used to get its status."""
-        return f'https://{self.config.harmony_hostname}/jobs/{job_id}'
+        return f'{self.config.root_url}/jobs/{job_id}?linktype={link_type.value}'
 
     def _cloud_access_url(self) -> str:
-        return f'https://{self.config.harmony_hostname}/cloud-access'
+        return f'{self.config.root_url}/cloud-access'
```

Comment:
Funny enough I did this exact same thing in the 721 branch.  Happy for the merge conflict if I can test locally.